examples/http-server/server/server.cpp

```diff
@@ -309,6 +259,11 @@ void on_sleep(const httplib::Request& request, httplib::Response& response) {
   }
 
   const std::string_view raw = begin->second;
+
+// NOTE(@dmehala): clang does not support std::from_chars with floating numbers
+#ifdef __clang__
+  double seconds = std::stod(std::string{raw});
+#else
```

Comment:
Find out specifically which clangs (maybe it's all of them) lack support for this, and update the comment. Then, remove the `from_chars` implementation and add error handling to whichever alternative implementation you choose (`stod` as here, or maybe `istringstream`).

Reply:
Resolved in 4c20da00573f307d44b3554528baf3c22727da06